packages/dapp/src/components/DepositFunds.jsx

```diff
@@ -219,6 +219,18 @@ export const DepositFunds = ({ invoice, deposited, due, tokenData }) => {
             <Text>{`${utils.formatUnits(deposited, decimals)} ${symbol}`}</Text>
           </VStack>
         )}
+        {deposited && (
+          <VStack align="flex-start">
+            <Text fontWeight="bold">Potential Dispute Fee</Text>
+            <Text>{`${(
+              utils.formatUnits(
+                amount.toNumber() - deposited.toNumber(),
+                decimals,
+              ) *
+              (1 / parseInt(invoice.resolutionRate))
```

Comment:
Rather than repeat this math, maybe move it to a utils library or something. It isn't super clear that resolution rate is 1/percent and putting it in the component hides this logic a bit that it could be misunderstood elsewhere.

Reply:
sounds good. yes, this does appear in quite a few places and it's not super intuitive how the percentage is calculated so I'll move it out